package.json

```diff
@@ -12,24 +12,38 @@
   "homepage": "http://locuszoom.org",
   "license": "MIT",
   "author": "Christopher Clark <chriclrk@umich.edu>",
+  "contributors": [
+    "Matthew Flickinger",
+    "Ryan Welch",
+    "Peter VandeHaar",
+    "Andy Boughton <abought@umich.edu>"
+  ],
   "main": "locuszoom.app.js",
+  "engines": {
+    "node": ">=6.12.0"
```

Comment:
**BLOCKER**
As mentioned elsewhere I had dependency failures on node >= `6.4.0`, but `5.12.0` worked. Maybe pin this to 5.x until we can sort out dev dependencies in a later iteration?